.github/workflows/cloud_regression.yml

```diff
@@ -1,9 +1,17 @@
 name: Trigger Cloud Regression E2E Tests
 on:
   push:
-    branches: [main]
-  pull_request:
-    branches: [main]
+    branches: [master]
+    paths:
+      - 'CMakeLists.txt'
+      - '**.c'
+      - '**.h'
+  pull_request_target:
+    branches: [master]
+    paths:
+      - 'CMakeLists.txt'
+      - '**.c'
+      - '**.h'
```

Comment:
Unless our goal is solely to test cloud integration, we probably want a larger list here.

Even if our goal is _just_ testing cloud integration though, we also should be triggering on any changes to the various ACLK related submodules (such as the `aclk/aclk-schemas` and `mqtt_websockets`), which will not match this.

Reply:
Thanks @Ferroin will add these as well. Is it just these 2? Are there any other?

Reply:
I _think_ there are others, but I’m not 100% certain. @underhood is probably the person to ask.

Reply:
ACLK related submodules... there are few of them but all under `mqtt_websockets` folder (it is a tree). For changes on those however we are interested whether the connection to cloud can be made and maintained. No need to go trough whole test suite.
Change to `mqtt_websockets` and its children may affect ability to maintain or establish connection. They will not affect any application layer messages or logic.

Reply:
Given that this is supposed to be triggering regression testing, I think we (probably) do want to run for changes there. A broken transport layer is still a regression, even if it’s not necessarily in the core agent code.

We definitely need to retest on changes to `aclk/aclk-schemas` though, that’s application-layer stuff.

Reply:
Thanks guys, so to make sure I understood correctly and apply the required changes. 

I should edit the list of tracked files to the following:


paths:
      - 'CMakeLists.txt'
      - '**.c'
      - '**.h'
      -'aclk/aclk-schemas/*'
      -'mqtt_websockets/*'